client/vite.config.js

```diff
@@ -0,0 +1,34 @@
+import { defineConfig } from 'vite'
+import solidPlugin from 'vite-plugin-solid'
+// import devtools from 'solid-devtools/vite';
+
+export default defineConfig({
+  plugins: [
+    solidPlugin()
+  ],
+  server: {
+    port: 3000,
```

Comment:
```suggestion
    port: 3000,
    watch: {
        usePolling: true
    },
```
This allows for autoreload of the frontend inside WSL2 for "windows" users who are editing the files outside of the WSL2 VM.
Check usePolling in https://vitejs.dev/config/server-options#server-watch

Reply:
done